app/controllers/atomic_admin/atomic_tenant_deployment_controller.rb

```diff
@@ -41,17 +35,20 @@ def search
       }
     end
 
-    # def index
-    #   page = AtomicTenant::LtiDeployment.all.order(:id).paginate(page: params[:page], per_page: 30)
-    #   render json: {
-    #     deployments: page,
-    #     page: params[:page],
-    #     total_pages: page.total_pages
-    #   }
-    # end
+    def index
+      page, meta = filter(
+        AtomicTenant::LtiDeployment.where(application_instance_id:),
+        search_col: "deployment_id",
+      )
+
+      render json: {
+        deployments: page,
+        meta:
+      }
+    end
```

Comment:
@blunckr-aj I think you touched some of the LTI deployment code recently & the search method above this is more complicated than I made this. What's the reasoning for the additional complexity?

Reply:
We discussed in slack, but the original version pulls in info from atomic_tenant that we don't think really needs to be here